raiden-cli/src/routes/channels.ts

```diff
@@ -111,7 +118,7 @@ async function openChannel(this: Cli, request: Request, response: Response, next
     } else if (isInsuficientFundsError(error)) {
       response.status(402).send(error.message);
     } else if (isConflictError(error)) {
-      response.status(409).send(error.message);
+      response.status(409).send({ message: error.message });
```

Comment:
We should maybe adapt to do so everywhere. At the moment it is an unhealthy mix of the old plain text and the new object version.

Reply:
yes, I agree.. right now, the CLI is a bit on reactive mode, fixing issues as they arise on the scenarios, but we should be more pragmatic about these groupings and return formats at some point.